src/cf-typography/usage.md

```diff
@@ -153,207 +185,79 @@ you should use the `aside` element.
 </aside>
 ```
 
+## Headings
 
-## Micro copy
-
-The `.micro-copy` class is good for highlighting small pieces of text.
-
-### Standard micro copy
-
-<p class="a-micro-copy">
-    Lorem ipsum dolor sit amet
-</p>
-
-```
-<p class="a-micro-copy">
-    Lorem ipsum dolor sit amet
-</p>
-```
-
-### Large micro copy
-
-<p class="a-micro-copy__large">
-    Lorem ipsum dolor sit amet
-</p>
-
-```
-<p class="a-micro-copy__large">
-    Lorem ipsum dolor sit amet
-</p>
-```
-
-## Date
-
-### Default date
-
-<span class="a-date">
-    Nov 4, 2013
-</span>
-
-```
-<span class="a-date">
-    Nov 4, 2013
-</span>
-```
-
+### Padded heading
 
-## Headers
+<h2 class="a-heading a-heading__padded">
+    Heading
+</h2>
 
-### Category slug
 
-The category slug is used on top of a section.
-Since categories can be repetitive,
-we suggest using a `.u-visually-hidden` element to add more context for screen readers.
+### Heading with icon
 
-#### Default category slug
+The heading with icon is typically used for listing categories in a meta header.
+Since categories can be repetitive, we suggest placing a label with `.u-visually-hidden`
+prior to the headings to add more context for screen readers (see Meta Header below).
 
-<a href="#" class="m-category-slug">
-    <span class="m-category-slug_icon cf-icon cf-icon-dialogue"></span>
-    <span class="u-visually-hidden">Category:</span>
```

Comment:
I think that's a template implementation detail that should be left up to the user. We want to encourage the use of the hidden text when there's only one category present.

To play devil's advocate with myself, if this is no longer called a "category slug" but is called a "Heading with icon", perhaps the example should be changed to something else and we can avoid this issue entirely.